manifests/init.pp

```diff
@@ -175,24 +177,19 @@
     validate_re($logdir,'^(\$(?!/)|/).+')
     validate_re($rundir,'^(\$(?!/)|/).+')
 
+    if $ca_crl_pull_interval != 2 {
+      deprecation('pupmod::ca_crl_pull_interval', 'pupmod::ca_crl_pull_interval is deprecated, the CRL cron job has been removed.')
```

Comment:
I think the deprecation is an important step for the removal here. but I will switch to `undef`